lego/apps/meetings/models.py

```diff
@@ -52,6 +52,7 @@ def get_reactions_grouped(self, user):
                     "count": 0,
                     "has_reacted": False,
                     "reaction_id": None,
+                    "user": user.full_name,
```

Comment:
I don't think this is the user you want. This will always be the currently logged in user.